general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -262,21 +262,19 @@ def resource_estimate(self, json_dict: Dict[str, str]) -> Dict[str, List[Dict[st
         """
         return self.post_request("/resource_estimate", json_dict)
 
-    def aqt_compile(self, json_dict: Dict[str, Union[int, str, List[str]]]) -> Dict[str, str]:
+    def aqt_compile(self, json_dict: Dict[str, Any]) -> Dict[str, str]:
```

Comment:
this shouldn't be necessary - the options dictionary is serialized prior to this so the annotation here shouldn't need to change
```suggestion
    def aqt_compile(self, json_dict: Dict[str, Union[int, str, List[str]]]) -> Dict[str, str]:
```
(ditto for other annotations in this module)

Reply:
Ah thanks, I missed that.

Reply:
since `request_json` in `service.py`'s `_aqt_compile()` is type `Dict[str,str]`